src/pybind/mgr/dashboard/controllers/ceph_users.py

```diff
@@ -12,13 +18,82 @@ class CephUserCaps(NamedTuple):
     mds: str
 
 
+class Cap(NamedTuple):
+    entity: str
+    cap: str
+
+
+def user_list(_):
+    """
+    Get list of ceph users and its respective data
+    """
+    return CephService.send_command('mon', 'auth ls')["auth_dump"]
+
+
+def user_create(_, user_entity: str, capabilities: List[Cap]):
+    """
+    Add a ceph user with its defined capabilities.
+
+    Example:
+        {
+            entity: "client.foo",
+            caps: [{entity: "mon", caps: "allow *"}]
+        }
+    """
+    # Caps are represented as a vector in mon auth add commands. Look at AuthMonitor.cc::valid_caps
+    # for reference.
+    caps = []
+    for cap in capabilities:
+        caps.append(cap['entity'])
+        caps.append(cap['cap'])
+
+    logger.debug("Sending command 'auth add' of entity '%s' with caps '%s'",
+                 user_entity, str(caps))
+    try:
+        CephService.send_command('mon', 'auth add', entity=user_entity, caps=caps)
+    except SendCommandError as ex:
+        msg = f'{ex} in command {ex.prefix}'
+        if ex.errno == -EINVAL:
+            raise DashboardException(msg, code=400)
+        raise DashboardException(msg, code=500)
+    return f"Successfully created user '{user_entity}'"
+
+
+create_cap_container = ArrayHorizontalContainer('Capabilities', 'capabilities',
+                                                label_html_class='hidden cd-header mt-1', fields=[
+                                                    FormField('Entity', 'entity',
+                                                              field_type=str, html_class='mr-3'),
+                                                    FormField('Entity Capabilities',
+                                                              'cap', field_type=str)
+                                                ])
+create_container = VerticalContainer('Create User', 'create_user',
+                                     html_class='d-none', fields=[
+                                         FormField('User entity', 'user_entity',
+                                                   field_type=str),
+                                         create_cap_container,
+                                     ])
```

Comment:
Because of json schema works, if you want to change the css of the form you have to modify the json schema itself. This was the closes 1:1 mapping of css to the form field.